tests/phpunit/Configuration/ConfigurationFactoryTest.php

```diff
@@ -936,7 +937,113 @@ public static function valueProvider(): iterable
                 'TrueValue' => new TrueValue(new TrueValueConfig([])),
             ])(),
             'phpspec',
-            'config/bootstrap.php',
+            __DIR__ . '/../Fixtures/Files/bootstrap/bootstrap.php',
+            '-d zend_extension=xdebug.so',
+            false,
+            '--stop-on-failure',
+            '/path/to/dist/coverage',
+            true,
+            true,
+            true,
+            true,
+            true,
+            72.3,
+            true,
+            81.5,
+            [],
+            false,
+            MapSourceClassToTestStrategy::SIMPLE,
+        ];
+
+        yield 'custom mutator with bootstrap file' => [
+            false,
+            false,
+            new SchemaConfiguration(
+                '/path/to/infection.json',
+                10,
+                new Source(['src/'], ['vendor/']),
+                new Logs(
+                    '/text.log',
+                    '/report.html',
+                    '/summary.log',
+                    '/json.log',
+                    '/gitlab.log',
+                    '/debug.log',
+                    '/mutator.log',
+                    true,
+                    StrykerConfig::forFullReport('master'),
+                    '/summary.json',
+                ),
+                'config/tmp',
+                new PhpUnit(
+                    'config/phpunit-dir',
+                    'config/phpunit',
+                ),
+                null,
+                null,
+                null,
+                ['@default' => false, 'CustomMutator' => true],
```

Comment:
here is a trick: we use `CustomMutator` alias, that is only available if `bootstrap` file is executed, making this alias available for class autoloader.

This confirms that custom mutator works with bootstrap